deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts

```diff
@@ -0,0 +1,107 @@
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import {
+  diffState,
+  getCometConfig,
+} from '../../../../plugins/deployment_manager/DiffState';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import {
+  calldata,
+  exp,
+  getConfigurationStruct,
+  proposal,
+} from '../../../../src/deploy';
+import { expect } from 'chai';
+
+export default migration('1687878801_increase-supply-caps', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (
+    deploymentManager: DeploymentManager,
+    govDeploymentManager: DeploymentManager,
+    vars: Vars
+  ) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      bridgeReceiver,
+      comet,
+      configurator,
+    } = await deploymentManager.getContracts();
+
+    const {
+      lineaMessageService,
+      governor,
+    } = await govDeploymentManager.getContracts();
+
+    const configuration = await getConfigurationStruct(deploymentManager);
+
+    const setConfigurationCalldata = await calldata(
+      configurator.populateTransaction.setConfiguration(
+        comet.address,
+        configuration
+      )
+    );
+    const l2ProposalData = utils.defaultAbiCoder.encode(
+      ['address[]', 'uint256[]', 'string[]', 'bytes[]'],
+      [
+        [configurator.address],
+        [0],
+        [
+          'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+        ],
+        [setConfigurationCalldata],
+      ]
+    );
+
+    const goerliActions = [
+      // 1. Set Comet configuration
+      {
+        contract: lineaMessageService,
+        signature: 'sendMessage(address,uint256,bytes)',
+        args: [bridgeReceiver.address, 0, l2ProposalData],
+      },
+    ];
+
+    const description =
+      '# Increase supply caps for Linea-Goerli cUSDCv3 market';
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(
+        await governor.propose(...(await proposal(goerliActions, description)))
+      )
+    );
+
+    const event = txn.events.find((event) => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return false;
+  },
+
+  async verify(
+    deploymentManager: DeploymentManager,
+    govDeploymentManager: DeploymentManager,
+    preMigrationBlockNumber: number
+  ) {
+    await deploymentManager.spider(); // We spider here to pull in Linea COMP now that reward config has been set
```

Comment:
I don't think this line is necessary in this situation

Reply:
Done